src/rendering/custom-element-widget.js

```diff
@@ -19,33 +19,31 @@ function makeDispatchFunction(element, eventName) {
 
 function subscribeDispatchers(element) {
   let {customEvents} = element.cycleCustomElementMetadata;
-  let disposables = new Rx.CompositeDisposable();
+  let dispatcherObservables = [];
   for (let streamName in customEvents) { if (customEvents.hasOwnProperty(streamName)) {
     if (/\$$/.test(streamName) &&
       streamName !== 'vtree$' &&
       typeof customEvents[streamName].subscribe === 'function')
     {
       let eventName = streamName.slice(0, -1);
-      let disposable = customEvents[streamName].subscribe(
+      let obs = customEvents[streamName].doOnNext(
```

Comment:
I refactored it to make the event-observables able to connect with `subscribeDispatchersWhenRootChanges`, and we could have a single subscription for all events. Right now I'm probably gonna revert this back because it's simpler to just have chaining subscriptions. However, it could make #110 harder to fix.